.travis.yml

```diff
@@ -9,6 +9,8 @@ matrix:
     - env: SCRIPT=lint VIM_VERSION=vim-8.0
 install:
   - ./scripts/install-vim $VIM_VERSION
+  - ./scripts/install-go-tools vim-8.0
```

Comment:
This needs to be `$VIM_VERSION`, otherwise it tries to use a non-existing Vim version.